db/seeds/cip_seed_dump.rb

```diff
@@ -666,3 +666,6 @@
   ["08fbf907-e6e0-4af2-8968-1f96e0ec4acf", "Lesson content", "## Time allocation\n\n5 minutes\n\n## Instructions\n\n- Use the notepad to reflect on your learning in this Block and further professional development.\n- Write just a couple of sentences for each question.\n\n$CTA\n\n<h2>In your notepad</h2>\n<ol>\n  <li style=\"font-weight: 400;\">\n    <span style=\"font-weight: 400;\">What have you learned in this Block?</span>\n  </li>\n  <li style=\"font-weight: 400;\">\n    <span style=\"font-weight: 400;\">What do you need to learn more about?</span>\n  </li>\n</ol>\n$CTA\n\n", nil, "9ed2ae28-4a0f-4a50-ae31-a17328f8517d"]
 ])
+CoreInductionProgramme.import([:id, :name, :course_year_one_id, :course_year_two_id], [
+  ["1ed4f9c6-3dbf-47f3-8371-e11a5424ffed", "Test Core induction programme", nil, nil]
+])
```

Comment:
Should this be here? I'm not sure what added it

Reply:
Kill it